client/client_grpc_options.go

```diff
@@ -29,6 +30,12 @@ func WithConsistencyLevel(cl entity.ConsistencyLevel) CreateCollectionOption {
 	}
 }
 
+func WithCollectionProperties(properties []*common.KeyValuePair) CreateCollectionOption {
+	return func(req *server.CreateCollectionRequest) {
+		req.Properties = properties
+	}
+}
```

Comment:
It's weird to expose `commonpb` defined type to users. Suggest to change signature to 
``` go
func WithcollectionProperties(key string, value string) CreateCollectionOption{
return func(req *server.CreateCollectionRequest) {
    req.Properties = append(req.Properties, &common.KeyValuePair{Key: key, Value: value})
}

```